src/openforms/js/components/form/select.js

```diff
@@ -1,5 +1,175 @@
-import {Formio} from 'formiojs';
-import {defineChoicesEditFormTabs} from './abstract';
+import {Formio, Utils} from 'formiojs';
+import DEFAULT_TABS, {ADVANCED, BASIC, REGISTRATION, VALIDATION} from './edit/tabs';
 
+const Select = Formio.Components.components.select;
 
-defineChoicesEditFormTabs(Formio.Components.components.select, 'data.values');
+const values = [{
+    type: 'datagrid',
+    input: true,
+    label: 'Values',
+    key: 'data.values',
+    tooltip: 'The radio button values that can be picked for this field. Values are text submitted with the form data. Labels are text that appears next to the radio buttons on the form.',
+    weight: 10,
+    reorder: true,
+    defaultValue: [{label: '', value: ''}],
+    components: [
+        {
+            label: 'Label',
+            key: 'label',
+            input: true,
+            type: 'textfield',
+        },
+        {
+            label: 'Value',
+            key: 'value',
+            input: true,
+            type: 'textfield',
+            allowCalculateOverride: true,
+            calculateValue: {_camelCase: [{var: 'row.label'}]},
+        },
+    ],
+}];
+
+
+const APPOINTMENT = {
+    key: 'appointment',
+    label: 'Appointment',
+    components: [
+        {
+            type: 'checkbox',
+            key: 'showProducts',
+            label: 'Show Products',
+            tooltip: 'Show products the user can book in this component'
+        },
+        {
+            type: 'checkbox',
+            key: 'showLocations',
+            label: 'Show Locations',
+            tooltip: 'Show locations for a given product in this component'
+        },
+        {
+            type: 'select',
+            input: true,
+            label: 'Select product component for locations',
+            key: 'productForLocations',
+            dataSrc: 'custom',
+            tooltip: 'Choose the product component and we wil prefill locations for that product in this component',
+            valueProperty: 'value',
+            data: {
+                custom(context) {
+                    return Utils.getContextComponents(context);
```

Comment:
can we filter this down to only `type: select` components?

Reply:
Not as far as I can tell.  I believe this is the function being called https://github.com/formio/formio.js/blob/c94992cacc50924d277132045dee0245a23e44b0/src/utils/utils.js#L1113-L1126 and it doesn't look like there is a way to either specify the type in the context or filter the values this function returns since it returns a list of objects with just a `label` and `value`.

If there is a way though it would be good to do :)